problems/1.intro.js

```diff
@@ -14,17 +14,17 @@ function helloWorld() {
  * @param {string} name - the name to be added to the console log 
  * @returns undefined
  */
-function helloName() {
-
+function helloName(name) {
+  return undefined
```

Comment:
This function should return "Hello {name}" as opposed to undefined.

Reply:
Thank you! For helping! I really appreciate it!